.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDateTimePeriodExtractorConfiguration.cs

```diff
@@ -21,13 +21,11 @@ public class SpanishDateTimePeriodExtractorConfiguration : BaseOptionsConfigurat
         private static readonly Regex BetweenRegex = new Regex(DateTimeDefinitions.BetweenRegex,
             RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
-        //TODO: add this according to the related part in English
         public static readonly Regex WeekDayRegex =
-            new Regex(@"^[\.]", RegexOptions.IgnoreCase | RegexOptions.Singleline);
+            new Regex(DateTimeDefinitions.WeekDayRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);

```

Comment:
Portuguese doesn't also have this issue?

Reply:
Fixed.